src/include/miopen/solver.hpp

```diff
@@ -2463,6 +2487,7 @@ struct GemmBwd1x1_stride1 : GemmBwdBase
     {
         return GetWorkspaceSize(ctx, ctx.conv_problem);
     }
+    bool MayNeedWorkspace() const { return true; }
```

Comment:
Wouldn't it be better to put MayNeedWorkspace() in the base class (GemmBwdBase)?